Uplift/Views/SportsForm/SportsFormBubbleCollectionViewCell.swift

```diff
@@ -0,0 +1,117 @@
+    private var identifier: String = ""
+    private var list: [BubbleItem] = []
+       
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        dropdownHeader.delegate = self
+        contentView.addSubview(dropdownHeader)
+           
+        divider.backgroundColor = .gray01
+        contentView.addSubview(divider)
+        
+        tableView.register(SportsFormBubbleTableViewCell.self, forCellReuseIdentifier: sportsFormBubbleListItemIdentifier)
+        tableView.register(SportsFormBubbleInputTableViewCell.self, forCellReuseIdentifier: sportsFormBubbleListInputIdentifier)
+        tableView.isScrollEnabled = false
+        tableView.separatorStyle = .none
+        tableView.backgroundColor = .clear
+        tableView.delegate = self
+        tableView.dataSource = self
+        contentView.addSubview(tableView)
+           
+        setupConstraints()
+    }
+    
+    func configure(for title: String, list: [BubbleItem], dropdownStatus: DropdownStatus, identifier: String) {
+        dropdownHeader.setTitle(title: title)
+        if dropdownStatus == .open {
+            dropdownHeader.rotateArrowDown()
+        } else {
+            dropdownHeader.rotateArrowUp()
+        }
+        
+        self.identifier = identifier
+        self.list = list
+        
+        self.tableView.reloadData()
+    }
+       
+    func setupConstraints() {
+        let dividerHeight = 1
+        let dropdownHeaderHeight = 55
+        
+        dropdownHeader.snp.makeConstraints { make in
+            make.leading.trailing.top.equalToSuperview()
+            make.height.equalTo(dropdownHeaderHeight)
+        }
+        
+        tableView.snp.makeConstraints { make in
+            make.top.equalTo(dropdownHeader.snp.bottom)
+            make.leading.trailing.bottom.equalToSuperview()
+        }
+        
+        divider.snp.makeConstraints { make in
+            make.leading.trailing.bottom.equalToSuperview()
+            make.height.equalTo(dividerHeight)
+        }
+    }
+       
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension SportsFormBubbleCollectionViewCell: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return list.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if indexPath.row == list.count - 1 {
+            let cell = tableView.dequeueReusableCell(withIdentifier: sportsFormBubbleListInputIdentifier, for: indexPath) as! SportsFormBubbleInputTableViewCell
+            let item = list[indexPath.row]
+            cell.configure(for: item)
+            return cell
+        }
+        let cell = tableView.dequeueReusableCell(withIdentifier: sportsFormBubbleListItemIdentifier, for: indexPath) as! SportsFormBubbleTableViewCell
+        let item = list[indexPath.row]
```

Comment:
very minor but can we just declare `let item` once since its the same in both cases